test/e2e/scaffold/scaffold.go

```diff
@@ -340,12 +350,12 @@ func (s *Scaffold) afterEach() {
 		}
 	}
 
-	err := k8s.DeleteNamespaceE(s.t, s.kubectlOptions, s.namespace)
-	assert.Nilf(ginkgo.GinkgoT(), err, "deleting namespace %s", s.namespace)
-
-	for _, f := range s.finializers {
-		f()
-	}
+	//err := k8s.DeleteNamespaceE(s.t, s.kubectlOptions, s.namespace)
```

Comment:
Why comment these codes?

Reply:
-_-||, This must be for debugging, please change it back.

Reply:
Yes, I comment these for debugging and forgot to uncomment them... sry